make/RunTests.gmk

```diff
@@ -539,12 +539,11 @@ define SetupRunGtestTestBody
 	  $$(if $$($1_PASSED), , $$(eval $1_PASSED := 0)) \
 	  $$(eval $1_SKIPPED := $$(shell $$(AWK) \
 	      '/YOU HAVE [0-9]+ DISABLED TEST/ { \
-	          if (match($$$$0, /[0-9]+/, arr)) { \
-	            print arr[0]; \
+	          if (match($$$$0, /YOU HAVE ([0-9]+) DISABLED TEST/, arr)) { \
+	            print arr[1]; \
 	            found=1; \
-	          } \
-	          if (!found) { print 0; } \
-	      }' \
+	          } } END \
+	          { if (!found) print 0; }' \
```

Comment:
From what I can tell, this is the relevant fix, moving the fallback 0 to the `END` block. I would suggest reformatting this for better readability. Probably something like this. The `END` keyword should be at the same level as the regex line matcher above, though the mix of indentation levels here is a bit confusing.
```suggestion
	          } \
	       } \
	       END { if (!found) print 0; }' \
```

Reply:
Thanks, END keyword has been change to at the same level as the regrex line